src/main/resources/db/changelog/20231101175700-add_owner_key_to_cp_anonymous_consumer.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.19.xsd">
+
+    <changeSet id="20230021144200-1" author="sbakaj">
+        <comment>
+            Add owner key to anonymous cloud consumer
+        </comment>
+
+        <addColumn tableName="cp_anonymous_cloud_consumers">
+            <column name="owner_key" type="VARCHAR(250)">
+               <constraints nullable="true" />
+            </column>
+        </addColumn>
+
+    </changeSet>
+
+</databaseChangeLog>
```

Comment:
Since we are using a feature branch, we can just add this column to the changelog where the `cp_anonymous_cloud_consumers` table is being created! This prevents another file from being created.

The changelog where `cp_anonymous_cloud_consumers` is being created is: 20230803000000-create-anonymous-consumers.xml

Reply:
done